src/global/reco/reco.cc

```diff
@@ -19,7 +20,6 @@
 #include "MatchClusters_factory.h"
 #include "ReconstructedElectrons_factory.h"
 #include "ReconstructedJets_factory.h"
-#include "algorithms/reco/ChargedParticleSelector.h"
 #include "extensions/jana/JChainMultifactoryGeneratorT.h"
 #include "extensions/jana/JOmniFactoryGeneratorT.h"
```

Comment:
```suggestion
#include "extensions/jana/JOmniFactoryGeneratorT.h"
#include "extensions/spdlog/SpdlogExtensions.h"
```
suggested in https://github.com/eic/EICrecon/actions/runs/7506420354/job/20437970525?pr=1217